src/main/java/org/spongepowered/api/map/MapView.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.map;
+
+import com.flowpowered.math.vector.Vector2i;
+import com.google.common.collect.ImmutableList;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.map.cursor.MapCursor;
+import org.spongepowered.api.map.cursor.MapCursorHandle;
+import org.spongepowered.api.map.cursor.MapCursors;
+import org.spongepowered.api.util.Identifiable;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * A representation of a specific map and all of it's properties.
+ */
+public interface MapView extends Identifiable {
+
+    /**
+     * Returns if vanilla's default handling of cursors with item frames and
+     * players should be used, if disabled all vanilla cursors are removed and
+     * plugins must manually update cursor positions.
+     *
+     * @return True when using vanilla handling, false otherwise
+     */
+    boolean isUsingVanillaCursors();
```

Comment:
I don't know about everyone else, but I would much prefer this be called `isUsingDefaultCursors()` or something. Vanilla is a bit confusing IMO as it could refer to SpongeVanilla or the base game.